### ISSUE TITLE
disable intellihide?

### ISSUE THREAD

**Issue by avidwriter** [optimisme/gnome-shell-simple-dock#3]
Would it be possible to add an option in tweak tool to disable the intellihide behavior?  otherwise great extension, thanks!

**Comment by GabMus** [comment]
I forked this project to kinda disable intellihide. It works but it's a bit clumsy from time to time.
I'll leave the link here: https://github.com/GabMus/gnome-shell-simple-dock
If you want to contribute, you're more then welcome, I'd really use a hand!

**Comment by optimisme** [comment]
Hi GabMus,

Thanks for forking 'simple-dock' source.

I fixed several bugs with the dock (theme background not properly shown,
signals not properly processed after overview, middle ellipse on large
titles, ...)

I also added the option to 'always hide' the dock and ignore 'intellihide'.

When accepted, the source will be on 'gnome extensions' for gnome 3.14 and
in several hours on Github for gnome 3.16.

If you want to update your version feel welcome to use my source.

Regards,

Albert
